assets/terraform-modules/packet/flatcar-linux/kubernetes/outputs.tf

```diff
@@ -35,10 +35,18 @@ output "calico_values" {
   value = module.bootkube.calico_values
 }
 
+output "calico-host-protection_values" {
+  value = local_file.calico_host_protection.*.content
```

Comment:
Otherwise it produces an array and we expect string.
```suggestion
  value = local_file.calico_host_protection.content
```

Reply:
Ah, Terraform. Some day I'll learn it.